src/js/signin/components/Verify.jsx

```diff
@@ -0,0 +1,83 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { handleVerify } from '../../common/helpers/login-helpers.js';
+
+class Verify extends React.Component {
+  componentDidMount() {
+    window.dataLayer.push({ event: 'verify-prompt-displayed' });
+  }
+
+  render() {
+    return (
+      <main className="verify">
+        <div className="row">
+          <div className="columns">
+            <div className="logo">
+              <a href="/">
+                <img alt="vets.gov" className="va-header-logo" src="/img/design/logo/logo-alt.png"/>
+              </a>
+            </div>
+          </div>
+        </div>
+        <div className="container">
+          <div className="row">
+            <div className="columns small-12">
+              <h1>Verify your identity</h1>
+            </div>
+          </div>
+          <div className="row hide-for-medium-up mobile-explanation">
```

Comment:
This shouldn't appear on the verify screen

<img width="341" alt="verify__vets_gov" src="https://user-images.githubusercontent.com/4106356/30386474-c5d560f6-985e-11e7-8e6c-1c16f2de8bd6.png">